scss/components/_table-styles.scss

```diff
@@ -0,0 +1,43 @@
+// Table styles
+
+.simple-table {
+  border-color: $primary;
+  border-style: solid;
+  border-width: 2px 0;
+  color: $primary;
+  font-family: $sans-serif;
+  margin: 0 0 2rem 0;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `0 0 2rem` instead of `0 0 2rem 0`